modules/ROOT/pages/_partials/security/index.adoc

```diff
@@ -8,7 +8,7 @@
 = Introduction to the {device-security-service} Service
 
 include::../generic-service-intros.adoc[tags=device-security-service]
+include::./ref_terminology.adoc[leveloffset=+1]
 
-[id='{context}_additional-resources-{context}']
```

Comment:
what are these tags used for?

Reply:
@finp

Reply:
from mod docs templates https://redhat-documentation.github.io/modular-docs/

Reply:
let's add it when we get to that step later?